### DISCUSSION TITLE
Making a drill session custom layout

### DISCUSSION THREAD

**Discussion by st4ycool** [ekiefl/pooltool#197] [Ideas]
Hey! I tried to make a drill layout to exercise but failed to find the right place to do it.
Could you please give a hint on how to do it most simply?

**Comment by ekiefl** [comment]
You'll have to be more specific.

**Comment by yuriianikin-nc** [comment]
Aim was to just put balls in a particular layout. E.g. six balls near each corner. 

And the level up would be to spawn a ball after each pot

**Comment by ekiefl** [comment]
Sorry, that's not easy right now.

You could make a specific layout with the infrastructure provided in `pooltool/layouts.py` and then make a specific game with the infrastructure provided in `pooltool/game/`. Finally, change line 543 of `pooltool/ani/animate.py` to select your new practice mode.
